sdk/src/clearingHouse.ts

```diff
@@ -1759,6 +1761,60 @@ export class ClearingHouse {
 		});
 	}
 
+	public async settleExpiredMarketPoolsToRevenuePool(
+		perpMarketIndex: number
+	): Promise<TransactionSignature> {
+		const marketAccountInfos = [];
+		const oracleAccountInfos = [];
+		const spotMarketAccountInfos = [];
+		const market = this.getPerpMarketAccount(perpMarketIndex);
+		marketAccountInfos.push({
+			pubkey: market.pubkey,
+			isWritable: true,
+			isSigner: false,
+		});
+		oracleAccountInfos.push({
+			pubkey: market.amm.oracle,
+			isWritable: false,
+			isSigner: false,
+		});
+
+		spotMarketAccountInfos.push({
+			pubkey: this.getSpotMarketAccount(QUOTE_SPOT_MARKET_INDEX).pubkey,
+			isSigner: false,
+			isWritable: true,
+		});
+
+		const remainingAccounts = oracleAccountInfos
```

Comment:
dont think you need to do the remainingAccounts since you dont load the maps in the smart contract ix?